### ISSUE TITLE
When Windows and NVDA magnifiers are both started, errors are logged and NVDA Magnifier commands report incorrect information

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20012]
### Steps to reproduce

* Start Windows Magnifier
* Start NVDA Magnifier without exiting Windows Magnifier
* Try to use NVDA Magnifier commands, e.g. paning, zoom out, etc.
* Quit NVDA Magnifier
* Quit Windows Magnifier
* Start NVDA Magnifier (without Windows Magnifier this time)
* Quit NVDA Magnifier

### Actual behavior
If both Windows and NVDA magnifiers are started, the log is spammed with errors.
We can see that NVDA Magnifier is in an endless process of updating error and NVDA magnifier attempting a recovery by restarting the magnification API again and again.

More, the zoomed view on the screen is not the one of NVDA Magnifier. But the magnifier commands report something that do not correspond to what is on the screen, e.g. modified zoom level.

### Expected behavior
If Windows magnification API cannot be used (e.g. due to Windows Magnifier already using it), just issue an error to inform the user when trying to start NVDA Magnifier.

I do not know why magnification API recovery was implemented in the first place. But we should at least have a limited time or number for recovery attempts.

If the magnification is not working, NVDA Magnifier commands should not act as if it was, e.g. report zoom level, etc.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/27057584/nvda.log)

#### NVDA installed/portable/running from source
Source
#### NVDA version
Commit b11969422d3ae15907b89ea5fdbcc262d5bce833 based on last master (commit 605fe47645ee12311e8949d53f7ccefac6fbb6d4).
Only French `gestures.ini` was changed.
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6783
#### Name and version of other software in use when reproducing the issue
N/A
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested

**Comment by CyrilleB79** [comment]
Cc @Boumtchack
